articles/mfa/guides/customize-mfa-universal-login.md

```diff
@@ -1,17 +1,38 @@
 ---
-title: Customize Multi-Factor Authentication
-url: /multifactor-authentication/custom
-description: Examples for configuring custom MFA implementations.
+description: Learn how to customize MFA pages with Universal Login branding options.
 topics:
   - mfa
   - custom-mfa
 contentType:
   - how-to
-  - concept
 useCase:
   - customize-mfa
 ---
-# Customize Multi-Factor Authentication
+# Customize MFA Pages with New Universal Login Branding Options
```

Comment:
This title only applies to the first section of the document, once it talks about "MFA API" then the title is not a good fit. The previous title was more ambiguous but worked better IMHO